app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
```

Comment:
you can unpack these variables in the easier way, also use the more descriptive names
command, old_file, new_file = command.split()